api/meter.go

```diff
@@ -42,6 +42,8 @@ type RelayMeter interface {
 	AllAppsLatencies(ctx context.Context) ([]AppLatencyResponse, error)
 	AllRelaysOrigin(ctx context.Context, from, to time.Time) ([]OriginClassificationsResponse, error)
 	RelaysOrigin(ctx context.Context, origin string, from, to time.Time) (OriginClassificationsResponse, error)
+
+	WriteHTTPSourceRelayCounts(ctx context.Context, counts []HTTPSourceRelayCount) error
```

Comment:
The `api` package should only serve relay metrics.
This needs a separate file under the collector package (which runs the http handler that we need for receiving data from TxDB)